usaspending_api/common/query_with_filters.py

```diff
@@ -310,7 +388,12 @@ class _RecipientScope(_Filter):
 
     @classmethod
     def generate_elasticsearch_query(cls, filter_value: str, query_type: _QueryType, **options) -> ES_Q:
-        recipient_scope_query = ES_Q("match", recipient_location_country_code="USA")
+        if query_type == _QueryType.SUBAWARDS:
+            recipient_scope_query = ES_Q("match", sub_legal_entity_country_code="USA") | ES_Q(
+                "match", sub_legal_entity_country_name="UNITED STATES"
```

Comment:
`sub_legal_entity_country_code` and `sub_legal_entity_country_name` aren't the names of the fields in Elasticsearch, they're `sub_recipient_location_country_code` and `sub_recipient_location_country_name`.

It looks like you used their database column names here.

Reply:
Good catch. Fixed